airflow/contrib/hooks/__init__.py

```diff
@@ -47,7 +47,8 @@
     'cloudant_hook': ['CloudantHook'],
     'fs_hook': ['FSHook'],
     'wasb_hook': ['WasbHook'],
-    'gcp_pubsub_hook': ['PubSubHook']
+    'gcp_pubsub_hook': ['PubSubHook'],
+    'aws_dynamodb_hook': ['AwsDynamoDBHook']
```

Comment:
`aws_dynamodb_hook`? The hook in this PR refers to the AWS firehose hook correct?